pkg/registry/apis/alerting/notifications/routing_tree/conversions.go

```diff
@@ -0,0 +1,202 @@
+package routing_tree
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/prometheus/alertmanager/config"
+	"github.com/prometheus/alertmanager/pkg/labels"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	promModel "github.com/prometheus/common/model"
+
+	model "github.com/grafana/grafana/pkg/apis/alerting_notifications/v0alpha1"
+	"github.com/grafana/grafana/pkg/services/apiserver/endpoints/request"
+	"github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
+	"github.com/grafana/grafana/pkg/util"
+)
+
+func convertToK8sResource(orgID int64, r definitions.Route, version string, namespacer request.NamespaceMapper) (*model.RoutingTree, error) {
+	spec := model.RoutingTreeSpec{
+		Defaults: model.RouteDefaults{
+			GroupBy:        r.GroupByStr,
+			GroupWait:      optionalPrometheusDurationToString(r.GroupWait),
+			GroupInterval:  optionalPrometheusDurationToString(r.GroupInterval),
+			RepeatInterval: optionalPrometheusDurationToString(r.RepeatInterval),
+			Receiver:       r.Receiver,
+		},
+	}
+	for _, route := range r.Routes {
+		if route == nil {
+			continue
+		}
+		spec.Routes = append(spec.Routes, convertRouteToK8sSubRoute(route))
+	}
+
+	var result = &model.RoutingTree{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       model.RouteResourceInfo.GroupVersionKind().Kind,
+			APIVersion: model.VERSION,
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            model.UserDefinedRoutingTreeName,
+			Namespace:       namespacer(orgID),
+			ResourceVersion: version,
+		},
+		Spec: spec,
+	}
+	result.SetProvenanceStatus(string(r.Provenance))
+	return result, nil
+}
+
+func convertRouteToK8sSubRoute(r *definitions.Route) model.Route {
+	result := model.Route{
+		GroupBy:           r.GroupByStr,
+		MuteTimeIntervals: r.MuteTimeIntervals,
+		Continue:          util.Pointer(r.Continue),
```

Comment:
when you make it optional in CUE, generator renders is as pointer. However, I thought it would not be correct to not return null if it's false. 
After you pointed this out, I think I would just make it regular boolean. In JSON it will drop `false` anyway.